### PR TITLE
Add cakephp 4.4 config set

### PR DESCRIPTION
**Pull request by othercorey**
This adds a config set for cakephp 4.4.

@markstory

### CODE REVIEW

**Review by samsonasik — APPROVED** [review]

File: config/sets/level/up-to-cakephp-44.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+declare(strict_types=1);
+
+use Rector\CakePHP\Set\CakePHPLevelSetList;
+use Rector\CakePHP\Set\CakePHPSetList;
+use Rector\Config\RectorConfig;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->sets([CakePHPSetList::CAKEPHP_44, CakePHPLevelSetList::UP_TO_CAKEPHP_43]);
+};
```
Comment (samsonasik) [inline config/sets/level/up-to-cakephp-44.php:11]: It need to be registered to `CakePHPLevelSetList` class constant

Comment (othercorey) [reply config/sets/level/up-to-cakephp-44.php:11]: Found it. Added.

**Review by samsonasik — APPROVED** [review]

**Comment by othercorey** [comment]
We had another last minute file move so I added it since this PR isn't merged yet.

**Review by samsonasik — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thank you :

**Comment by samsonasik** [comment]
@TomasVotruba mergeable?

**Pull request merged**